### ISSUE TITLE
Set dachs:latest to auto-start postgres/dachs

### ISSUE THREAD

**Issue by chbrandt** [gavodachs/docker-dachs#16]
Currently, during initialization, [dachs](https://github.com/gavodachs/docker-dachs/blob/master/dockerfiles/dachs/Dockerfile) just prints a _help_ message and keeps waiting for users to start/run the services on their own (https://github.com/gavodachs/docker-dachs/blob/40cc3874845dc2395a610c30a04441389af83030/dockerfiles/dachs/Dockerfile#L91).

Suce behavior is welcome when a simple `docker run` is used, but we can do better -- _ie_, start the services automatically -- when using docker-compose (through _command_).

**Comment by kidpixo** [comment]
We can specify a different entrypoint from the  [dachs/Dockerfile#L91](https://github.com/gavodachs/docker-dachs/blob/40cc3874845dc2395a610c30a04441389af83030/dockerfiles/dachs/Dockerfile#L91) in [docker-compose.yml#L14](https://github.com/gavodachs/docker-dachs/blob/master/docker-compose.yml#L14).

This way docker-compose starts dachs automagically, but running the image alone makes possible to validate & ingest the data.

**Comment by chbrandt** [comment]
Good point @kidpixo . Do you happen to have a couple of cycles to do it? 🤞
